src/Nethermind/Nethermind.Merge.Plugin.Test/EngineModuleTests.V1.cs

```diff
@@ -1661,6 +1663,46 @@ public async Task payloadV1_latest_block_after_reorg()
         }
     }
 
+    [Test]
+    public async Task Should_return_capabilities()
+    {
+        using var chain = await CreateBlockChain();
+        var rpcModule = CreateEngineModule(chain);
+        var expected = new[]
+        {
+            "engine_exchangeTransitionConfigurationV1",
+            "engine_executionStatus",
```

Comment:
And the value of these tests has been already proved
"engine_executionStatus"
This method can be removed

Reply:
and even it should be removed from code

The method is not needed now

Reply:
Alright, not in this PR though.